<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="c0016390" failures="25" tests="43" name="Rebill Test" time="4121.614" errors="0" timestamp="2021-05-05T22:44:52 UTC">
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod3" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod3() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod3() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod3 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod2" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod2() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod2() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod4" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod4() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod4() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod4 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="67.918">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.987">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="156.496"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.249">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="60.777">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="152.572"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="152.786"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="60.444">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="152.199"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="154.309"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="62.213">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="62.38">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="62.947">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.556">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="152.57"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.833">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.878">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="60.568">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="60.426">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="154.315"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="60.613">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="153.484"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="151.824"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="154.422"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="62.323">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="67.035">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.533">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="58.954">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="60.505">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="150.526"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="151.09"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.026">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="150.336"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="153.283"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="150.964"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="60.892">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="155.405"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="149.869"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="61.365">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="152.601"/>
</testsuite> <!-- Rebill Test -->
